src/components/TopBar.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import styled from 'styled-components';
+import { AiOutlineSearch } from 'react-icons/ai';
+function TopBar({ current, onSearchButtonClicked }) {
+  const menu = { friends: '친구', chat: '채팅', settings: '설정' };
+  return (
+    <>
+      <Wrapper>
+        <Title>{menu[current]}</Title>
+        {current === 'friends' || current === 'chat' ? (
+          <AiOutlineSearch
+            size={30}
+            onClick={onSearchButtonClicked}
+          ></AiOutlineSearch>
+        ) : (
+          ''
+        )}
```

Comment:
```suggestion
        {(current === 'friends' || current === 'chat') && (
          <AiOutlineSearch
            size={30}
            onClick={onSearchButtonClicked}
          />
        )}
```

이 부분도 이렇게 해볼수 있겠네요!
그리고 self-closing이 가능한 컴포넌트는 최대한 그렇게 하는것이 가독성에 유리하지 않나 싶습니다.